src/poetry/__init__.py

```diff
@@ -1 +1,4 @@
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore[has-type]
+__path__ = __import__("pkgutil").extend_path(
+    __path__,  # type: ignore[has-type]
+    __name__,
+)
```

Comment:
I'm afraid this change still makes some trouble. For example PyCharm isn't able to follow `poetry.core` imports.

I vague remember that this code line is quite critical for pkgutils namespace packages. But I don't understand why exactly :man_shrugging: So I'm afraid we have to make sure that this stays a one-liner and is the first line of this file (and can than remove the `namespace_packages` setting for isort again). :(

Reply:
Funny how much trouble a small, seemingly innocuous change can cause :D I have added a new commit reverting the change, and just slapped on a `  # noqa: E501` here.